src/__tests__/test-utils-wrapper.test.tsx

```diff
@@ -0,0 +1,99 @@
+
+function getComponentsList() {
+  const exceptions: Record<string, string> = {
+    AnnotationContext: 'Annotation',
+  };
+
+  return Object.keys(components)
+    .map(component => exceptions[component] ?? component)
+    .filter(Boolean);
+}
+
+describe('Test utils element wrapper', () => {
+  describe('Snapshot tests', () => {
+    const importPaths = [
+      {
+        type: 'dom',
+        relativePath: '../../lib/components/test-utils/dom/index.js',
+      },
+      {
+        type: 'selectors',
+        relativePath: '../../lib/components/test-utils/selectors/index.js',
+      },
+    ] as const;
+
+    test.each(importPaths)('$type wrapper stays unchanged', ({ relativePath }) => {
+      const testUtilsPath = path.join(__dirname, relativePath);
+      const domWrapper = fs.readFileSync(testUtilsPath, 'utf8');
+      expect(domWrapper).toMatchSnapshot();
+    });
+  });
+
+  describe.each(getComponentsList())('<%s />', componentName => {
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+
+    const testUtilTypes = [
+      {
+        type: 'dom',
+        testWrapper: new dom.ElementWrapper(document.createElement('div')),
+        testUtilImport: dom,
+      },
+      {
+        type: 'selector',
+        testWrapper: new selectors.ElementWrapper('div'),
+        testUtilImport: selectors,
+      },
+    ] as const;
+
+    describe.each(testUtilTypes)('$type wrapper', ({ testWrapper, testUtilImport }) => {
+      const componentNamePluralized = pluralize(componentName);
+
+      test(`find${componentName} calls findComponents with ${componentName}'s root selector`, () => {
+        const WrapperClass = testUtilImport[`${componentName}Wrapper` as 'AlertWrapper'];
+
+        testWrapper[`find${componentName}` as 'findAlert']();
+
+        expect(testUtilImport.ElementWrapper.prototype.findComponent).toHaveBeenCalledWith(
+          `.${WrapperClass.rootSelector}`,
+          WrapperClass
+        );
+      });
+
+      test(`findAll${componentNamePluralized} calls findAllComponents with ${componentName}'s root selector`, () => {
+        const WrapperClass = testUtilImport[`${componentName}Wrapper` as 'AlertWrapper'];
+
+        testWrapper[`findAll${componentNamePluralized}` as 'findAllAlerts']();
+
+        expect(testUtilImport.ElementWrapper.prototype.findAllComponents).toHaveBeenCalledWith(
+          `.${WrapperClass.rootSelector}`,
+          WrapperClass
+        );
+      });
+
+      test(`find${componentName}ByTestId calls findComponent with data-testid appended to ${componentName}'s root selector`, () => {
+        const WrapperClass = testUtilImport[`${componentName}Wrapper` as 'AlertWrapper'];
+
+        testWrapper[`find${componentName}ByTestId` as 'findAlertByTestId']('test-id');
+
+        expect(testUtilImport.ElementWrapper.prototype.findComponent).toHaveBeenCalledWith(
+          `.${WrapperClass.rootSelector}[data-testid="test-id"]`,
+          WrapperClass
+        );
+      });
```

Comment:
```js
test('selector utility works', () => {
   const selector = createSelectorWrapper().findAllAlerts().get(1).toSelector()
   expect(document.querySelector(selector)).toBeInDocument();
});
```

Reply:
The latest change includes this suggestion.